src/AcceptanceTests/TestSuiteConstraints.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using AcceptanceTesting.Support;
+
+    public partial class TestSuiteConstraints
+    {
+        public bool SupportsDtc { get; } = false;
+        public bool SupportsCrossQueueTransactions { get; } = true; // TODO: verify that this is an AtomicSendWithReceive. If not, set to false
```

Comment:
@timbussmann this is misleading then. Look at the links above.  The flag is only used in `When_receiving_with_native_multi_queue_transaction` which specifies `TransportTransactionMode.ReceiveOnly`